### PR TITLE
UI : Colored Backgrounds for Alert States #106

### PR DESCRIPTION
**Pull request by pranavk1511**
# Description

Summarize the change.
Implemented Alerts for Different states. 

Fixes #<106> ( UI: Colored Backgrounds for Alert States )

Made code changes for the row definition of the alert grid 

# Testing
Describe how you tested this code. How can the reviewers reproduce your tests?

1. Create a reg alert and trigger data for different states.  or 

`const alert state = hardcode the value here`  on all-alerts.js:137

# Checklist:
Before marking your pull request as ready for review, complete the following.

- [x] I have self-reviewed this PR.
- [x] I have removed all print-debugging and commented-out code that should not be merged.
- [x] I have added sufficient comments in my code, particularly in hard-to-understand areas.
- [x] I have formatted the code, if applicable. For Go, I have run `goimports -w .`.

### CODE REVIEW

File: siglens.db
Comment (AndrewHess) [inline siglens.db:1]: siglens.db shouldn't be version controlled. Please remove it from the PR

Comment (pranavk1511) [reply siglens.db:1]: Done .

File: static/js/all-alerts.js [hunk 1]
```diff
@@ -131,7 +131,31 @@ let alertColumnDefs = [
...
-        width:50,
+        width: 50,
+        cellStyle: (params) => {
+            console.log(params.data)
+            const alertState = params.data.alertState
+            console.log(alertState); 
+            let backgroundColor;
+            switch (alertState) {
+                case "Normal":
+                    backgroundColor = "#53DB6E"; // Green background for Normal State
```
Comment (sunitakawane) [inline static/js/all-alerts.js:142]: we should use css file to store the color values. @sonamgupta21 can you check the color palate for this

File: static/js/alert.js [hunk 2]
```diff
@@ -353,6 +353,7 @@ function displayAlertProperties(res) {
     const queryParams = res.queryParams;
     $('.alert-name').text(alertInfo.alert_name);
     $('.alert-status').text(mapIndexToAlertState.get(alertInfo.state));
+    $('.alert-status').text(alertStatus).css('background-color', backgroundColor);
```
Comment (sunitakawane) [inline static/js/alert.js:356]: add clss to alert-status based on alertInfo.state and then use css to set the color.

File: static/js/all-alerts.js (hunk 1 above)
Comment (pranavk1511) [reply static/js/all-alerts.js]: Updated the code according to instruction .

**Review by sonamgupta21 — COMMENTED** [review]
image
Please ensure that the CSS style matches the image provided in the issue, including text color and also the background color is only for the text, not for the whole cell.

**Comment by nbhavana** [comment]
@pranavk1511 any update for this PR?

**Comment by karpallavi** [comment]
I want to work on it.

**Comment by nkunal** [comment]
> I want to work on it.

Excellent! go ahead please

**Comment by AndrewHess** [comment]
Closing since there's been no recent updates

**Pull request closed without merging**